src/Containers/CustomArrayList.java

```diff
@@ -0,0 +1,50 @@
+package Containers;
+
+import java.util.Arrays;
+
+public class CustomArrayList<T> {
+    private Object[] elements;
+    private int size;
+
+    public CustomArrayList() {
+        elements = new Object[10]; // Initial capacity
```

Comment:
don't use magic numbers like 10 here, instead add field default capacity with this value